libraries/testkit/sgaccel.py

```diff
@@ -39,16 +41,24 @@ def stop(self):
 
     def start(self, config):
         conf_path = os.path.abspath(config)
-
         log.info(">>> Starting sg_accel with configuration: {}".format(conf_path))
 
+        playbook_vars = {
+            "sync_gateway_config_filepath": conf_path,
+            "server_port": self.server_port,
+            "server_scheme": self.server_scheme
+        }
+        if is_cbs_ssl_enabled(self.cluster_config) and get_sg_version(self.cluster_config) >= "1.5.0":
```

Comment:
This is a string comparison. compare_versions(version_one, version_two): in keywords.utils might be a better fit.